src/main/java/com/synopsys/integration/alert/channel/email/EmailChannel.java

```diff
@@ -98,8 +100,16 @@ public void distributeMessage(DistributionEvent event) throws IntegrationExcepti
 
     public void sendMessage(EmailProperties emailProperties, Set<String> emailAddresses, String subjectLine, String formatType, EmailAttachmentFormat attachmentFormat, MessageContentGroup messageContent) throws IntegrationException {
         String topicValue = null;
+        Optional<String> subTopicValue = Optional.empty();
         if (!messageContent.isEmpty()) {
             topicValue = messageContent.getCommonTopic().getValue();
+            subTopicValue = messageContent.getSubContent()
+                                .stream()
+                                .map(ProviderMessageContent::getSubTopic)
+                                .flatMap(Optional::stream)
+                                .filter(linkableItem -> MessageBuilderConstants.LABEL_PROJECT_VERSION_NAME.equals(linkableItem.getName()))
+                                .map(LinkableItem::getValue)
+                                .findFirst();
```

Comment:
Because there can be multiple project-versions, we should return a joined list of them.  

```suggestion
                                .collect(Collectors.joining(", "));
```

Reply:
In my testing of this feature I saw that for multiple versions we have multiple emails for each. So "projectA version1" and "projectA version2" would come in as 2 separate emails with those titles. Is there a case where they would be combined?

Reply:
I believe they are supposed to be combined if they get processed in the same batch of notifications.

Reply:
Either way, this will work.